inc/priam/cluster.hpp

```diff
@@ -107,6 +107,18 @@ class cluster
         std::chrono::milliseconds update_rate,
         uint64_t                  min_measured) -> bool;
 
+
+    /**
+     * Enables constant speculative executions for requests sent to the Cluster.
+     * @param delay Controls the length of time before sending a speculative request
+     * @param max_executions The maximum number of speculative requests to send.
+                             This should not be higher than your max replication factor.
+     * @return True if the heartbeat intervals were updated.
+     */
+    auto cluster::speculative_execution(
```

Comment:
shouldn't need the namespace in the header declaration

Reply:
That was a copy/paste fail. Fixed.